chapters/inheritance.tex

```diff
@@ -569,6 +569,27 @@ \subsection{Final Element Modification Prevention}\label{final-element-modificat
 end Test;
 \end{lstlisting}
 \end{example}
+If a class is final it means that there cannot be modifiers for any of its elements when using the class, i.e., it as if all elements were modified to be final.
```

Comment:
Strange wording: _it as if_

Reply:
It would also be better to give the rule in a direct style rather than in "as if" style:
```suggestion
Declaring a class final makes all its elements final.  In particular, it means that there cannot be modifiers for any of its elements when using the class.
```

What happens to replaceable classes in the final class?